src/main/scala/com/microsoft/hyperspace/index/sources/default/DefaultFileBasedRelation.scala

```diff
@@ -42,13 +42,12 @@ class DefaultFileBasedRelation(spark: SparkSession, override val plan: LogicalRe
   /**
    * Computes the signature of the current relation.
    */
-  override def signature: String = plan.relation match {
+  override def signature(fb: FingerprintBuilder): Option[Fingerprint] = plan.relation match {
```

Comment:
Ditto - update scaladoc explaining the new `fb` parameter.

Reply:
Done